test/resources/boltstub/hello_goodbye.script

```diff
@@ -0,0 +1,6 @@
+!: BOLT 3
+
+C: HELLO {"credentials": "password", "scheme": "basic", "user_agent": "single-connection/1.2.3", "principal": "neo4j"}
+S: SUCCESS {"server": "Neo4j/9.9.9"}
+C: GOODBYE
+S: SUCCESS {}
```

Comment:
Since actual server implementation doesn't send a `SUCCESS` message upon `GOODBYE` and stops the connection, this line should be replaced by an `EXIT` command?